.nvmrc

```diff
@@ -0,0 +1 @@
+lts/*
```

Comment:
@trickpattyFH20 let's replace this with the specific latest lts `lts/carbon` to be sure we're all on the same version.